src/Service/IftttHandlerTelegram.php

```diff
@@ -54,7 +54,7 @@ public function handleEventAnnouncement(Event $event)
         $imgUrl = $this->generateImageUrl($event);
         $body = json_encode(
             [
-                "value1" => nl2br($this->createEventAnnouncement($event)),
+                "value1" => nl2br($event->createEventAnnouncement()),
```

Comment:
@daria-sieroshtan 
:+1:  for moving this to entity
But within entity "event" you don't need to mention "event" in method name